src/components/PlaceholderContainer/__stories__/PlaceholderContainer.stories.tsx

```diff
@@ -0,0 +1,266 @@
+            content size for displaying very long texts.
+        </p>
+    </div>
+);
+
+const actionComponentTest = (
+    <div className={b('custom-action')}>
+        <DropdownMenu
+            defaultSwitcherProps={{view: 'flat-secondary'}}
+            items={[
+                {text: 'text 1', action: () => {}},
+                {text: 'text 2', action: () => {}},
+            ]}
+            onSwitcherClick={(e) => e?.stopPropagation()}
+            switcher={
+                <Button>
+                    Text
+                    <Icon data={ChevronDown} size={16} />
+                </Button>
+            }
+        />
+    </div>
+);
+
+const actionMainProps: PlaceholderContainerActionProps = {
+    text: 'Main button',
+    view: 'normal',
+    onClick: () => alert('Click by main button'),
+};
+
+const actionAdditionalBtnProps: PlaceholderContainerActionProps = {
+    text: 'Additional button',
+    view: 'flat-secondary',
+    onClick: () => alert('Click by additional button'),
+};
+
+const baseProps = {
+    title: 'Container with one button & image component',
+    image: <ImageComponentTest />,
+    className: 'placeholder-container',
+};
+
+const placeholderContainerProps: Omit<PlaceholderContainerProps, 'size' | 'direction'> = {
+    ...baseProps,
+    actions: [actionMainProps],
+    align: 'center',
+};
+
+const actionsProps = {
+    actions: [actionMainProps, actionAdditionalBtnProps],
+};
+
+const placeholderContainerCustomRenderedProps: Omit<
+    PlaceholderContainerProps,
+    'size' | 'direction'
+> = {
+    ...placeholderContainerProps,
+    content: contentComponentTest,
+};
+
+const placeholderContainerCustomActionProps: Omit<PlaceholderContainerProps, 'size' | 'direction'> =
+    {
+        ...placeholderContainerProps,
+        actions: actionComponentTest,
+    };
+
+const descriptionProps = {
+    description:
+        'Some long descriptionProps text that can contain of long long very long text etc. It can be repeated like this. Some long descriptionProps text that can contain of long long very long text etc.',
+    promoDescription:
+        "Comparing to 'L' size promo size has full width of the content block, a larger title size and alignment",
+};
+
+export const Size: Story = {
+    render: () => (
+        <React.Fragment>
+            <Showcase title="Size row">
+                <ShowcaseItem title="Size s">
+                    <PlaceholderContainer
+                        {...placeholderContainerProps}
+                        description={descriptionProps.description}
+                        direction="row"
+                        size="s"
+                    />
+                </ShowcaseItem>
+                <ShowcaseItem title="Size m">
+                    <PlaceholderContainer
+                        {...placeholderContainerProps}
+                        description={descriptionProps.description}
+                        direction="row"
```

Comment:
`direction` cases should be covered in its own story, same is for `align`. So basically there should be at least 5 stories: Default, Size, Direction, Align, Actions.